isthmus/src/main/java/io/substrait/isthmus/expression/ScalarFunctionConverter.java

```diff
@@ -24,13 +24,17 @@ public class ScalarFunctionConverter
         Expression,
         ScalarFunctionConverter.WrappedScalarCall>
     implements CallConverter {
-
   /**
    * Function mappers provide a hook point for any custom mapping to Substrait functions and
    * arguments.
    */
   private final List<ScalarFunctionMapper> mappers;
 
+  public enum INDEXING {
+    ONE,
+    ZERO
+  }
```

Comment:
Since this enum is used across multiple classes, I am not sure being a nested class within ScalarFunctionConverter is really the right place for it. This couples several other classes to ScalarFunctionConvert that previously did not reference it. Perhaps it should be declared in its own file. Maybe it is `ExtractIndexing` since I don't see it used anywhere else?

Reply:
I wasn't completely sure where best to put this - as it's only (at the moment anyway) used briefly;

I'll move it to a separate file